ui/app/pages/home/index.scss

```diff
@@ -124,4 +124,13 @@
     height: 100%;
     width: 100%;
   }
+
+  &__support {
+    padding: 10px 0 20px 0;
+    text-align: center;
+
+    a {
+      color: $Blue-500;
```

Comment:
[REQ] Instead of $Blue-500 could we use $primary-1 -- its the same value but matches the primary theme variables. Sorry its confusing :(